plexapi/video.py

```diff
@@ -323,7 +323,7 @@ def isWatched(self):
 
     def seasons(self, **kwargs):
         """ Returns a list of :class:`~plexapi.video.Season` objects. """
-        key = '/library/metadata/%s/children' % self.ratingKey
+        key = '/library/metadata/%s/children?excludeAllLeaves=1' % self.ratingKey
```

Comment:
Whats the benefit by excludeAllLeaves? Faster?

Reply:
Without it we'd have "All Seasons" as one more seasons, I think it's a bit unexpected.

Reply:
Im not sure i understand. This is supposed to returns all seasons. does that still happen?

Reply:
Yep, sure. If the show has 2 seasons it'll return [<Season 1>, <Season 2>], but if you'd omit `excludeAllLeaves` you'll receive [<All Seasons>, <Season 1>, <Season 2>].